### PR TITLE
Simple version of Facebook instant articles

### PR DESCRIPTION
**Pull request by gklopper**
Really simple. Just handles Video and Gallery at the moment. Still totally in test phase hence hardcoded list of 2 pieces of content.

This is what a gallery looks like in the test app.

### CODE REVIEW

**Comment by janua** [comment]
Can we namespace the views?
Also, I'm sure we are going to make a nice case class for the RSS eventually?

:+1:

**Comment by gklopper** [comment]
Won't be merging this immediately. Just wanted everyone to see what I was up to.

**Comment by gklopper** [comment]
I'll probably template the RSS rather than case class it.

**Comment by janua** [comment]
You could do both. Just don't touch `model.Content` anymore than we have to!

File: rss/app/instant/InstantArticlesRss.scala [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          </description>
+          <language>en-us</language>
+          <lastBuildDate>{DateTime.now.toISODateTimeNoMillisString}</lastBuildDate>
+          {content.map { item =>
+              <item>
+                <title>{PCData(item.webTitle)}</title>
+                <link>{item.webUrl}</link>
+                <guid>{item.guid}</guid>
+                <pubDate>{item.webPublicationDate.toISODateTimeNoMillisString}</pubDate>
+                <author>{PCData(item.contributors.map(_.name).mkString(", "))}</author>
```
Comment (JustinPinner) [inline rss/app/instant/InstantArticlesRss.scala]: Multiple authors require multiple <author> elements.

File: rss/app/instant/InstantArticlesRss.scala [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    // https://developers.facebook.com/docs/partners/native-articles
+    contentToIndex.map{ content =>
+      <rss version="2.0" xmlns:content="http://purl.org/rss/1.0/modules/content/">
+        <channel>
+          <title>The Guardian</title>
+          <link>http://www.theguardian.com</link>
+          <description>
+            Latest news, sport, business, comment, analysis and reviews from the Guardian, the world's leading liberal voice.
+          </description>
+          <language>en-us</language>
```
Comment (JustinPinner) [inline rss/app/instant/InstantArticlesRss.scala]: en-gb?

File: rss/app/views/content/main.scala.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    </head>
+    <body>
+        <article>
+            <header>
+
+                <h1>@Html(content.headline)</h1>
+                <details>
+                    <summary>@Html(content.description.getOrElse(""))</summary>
+                </details>
+                <time datetime="@content.webPublicationDate.toISODateTimeNoMillisString" class="op-published">December 11th, 4:44 PM</time>
```
Comment (JustinPinner) [inline rss/app/views/content/main.scala.html]: December 11th?

**Comment by JustinPinner** [comment]
Nice. But I seem to have put on my pedantic hat :tophat: Sorry

File: rss/app/views/content/main.scala.html (hunk 3 above)
Comment (gklopper) [inline rss/app/views/content/main.scala.html]: yes from now on all content is published then :scream:

Comment (paperboyo) [inline rss/app/views/content/main.scala.html]: > December 11th?

Not December 16th?! https://github.com/guardian/frontend/issues/8810 :laughing:

**Comment by gklopper** [comment]
This has run its course

**Pull request closed without merging**